sympy/geometry/tests/test_polygon.py

```diff
@@ -228,6 +228,11 @@ def test_polygon():
     assert intersection(m[p1], m[p2], m[p3]) == [t1.centroid]
     assert t1.medial == Triangle(Point(2.5, 0), Point(0, 2.5), Point(2.5, 2.5))
 
+    # Nine-point circle
+    vo_middles = [Segment(v, t1.orthocenter).midpoint for v in t1.vertices]
```

Comment:
@leosartaj Yeah, since `t1` is both right and isosceles, 9 points are degenerated to only 4 distinct.